pkg/rtc/signalhandler.go

```diff
@@ -56,6 +56,10 @@ func HandleParticipantSignal(room types.Room, participant types.LocalParticipant
 			pLogger.Warnw("could not update subscription", err,
 				"tracks", msg.Subscription.TrackSids,
 				"subscribe", msg.Subscription.Subscribe)
+		} else {
+			pLogger.Infow("updated subscription",
+				"tracks", msg.Subscription.TrackSids,
+				"subscribe", msg.Subscription.Subscribe)
```

Comment:
Going to keep this as it is useful to see when subscriptions were enabled/disabled.